handbook/ce/engaging-other-teams.md

```diff
@@ -1,5 +1,169 @@
+
+### Browser extension and code host native integrations
+
+**Keywords**: `browser extension`, `native integration`, `chrome`, `firefox`, `safari`, `Phabricator`, `Bitbucket`, `GitHub`, `GitLab`
+
+Any questions about the browser extension or code host native integrations should be routed to the [Web team](../engineering/web/index.md).
+
+### Sourcegraph extensions
+
+**Keywords**: `extensions`, `plugins`, `blame`, `git-extras`, `Codecov`
+
+Any questions about Sourcegraph extensions should be routed to the [Web team](../engineering/web/index.md).
+
+### Campaigns
+
+**Keywords**: `campaigns`, `large scale code changes`, `refactoring`, `src-cli`
+
+Any questions about campaigns should be routed to the [Campaigns team](../engineering/campaigns/index.md).
+
+
+## Step 4: File a Github issue
+
+It can seem duplicative, but it's just as important when we need help as it is when we file a defect to file a Github issue. This allows us to understand how we are doing toward our goal of being a self-sustaining team, as well as trends over time that impact the customer experience and our team's experience.
+
+### Select the right issue tracker
+
+We maintain two issue trackers:
+
+1. [sourcegraph/customer](https://github.com/sourcegraph/customer/issues) for private issues that affect a particular customer and our requests for help (RFHs).
+2. [sourcegraph/sourcegraph](https://github.com/sourcegraph/sourcegraph/issues) for public issues affecting the general product.
+
+When in doubt, file in the private repo—issues can be moved over to the public repo if needed. 
+
+File new customer issues in the [private customer issue tracker](https://github.com/sourcegraph/customer/issues/new):
+
+* Initiate the issue from Zendesk via the integration with Github so that the conversation with the customer is available to engineering directly in Github
+* Provide all information required for troubleshooting that you gathered in the previous steps. 
+* Label it with `customer/$name` and `rfh`. 
+* Assign the issue to the appropriate team you identified in the previous step. 
+
+If it turns our to be a general issue affecting multiple deployments, create an issue in the [public issue tracker](https://github.com/sourcegraph/sourcegraph/issues/new/choose). The issue must not include any private information. It is okay to link any relevant private, customer specific issues. If not already linked to the relevant Zendesk tickets, do so.
+
+### Add the priority label
+Add a prioritization label to the issue, from `p0` to `p3` based on a combination of (1) the severity of the issue, (2) the prioritization of the reporting company and (3) any relevant context that translates to level of urgency:
+
+* `p0`: System outages (`about`, `/search` (for sourcegraph.com only), or `docs` is fully unreachable) are always p0. At your discretion, we may also designate an issue p0 that results in the company's Sourcegraph instance being unusable. 
```

Comment:
We chatted about this in passing and I believe this is a future state plan for you but just to make sure—would it be possible to add some additional prioritization info around deployments specifically? (These definitions are pretty oriented towards existing customer SLAs, when a lot of our issues happen during deployment when no SLA is in place.)

Reply:
Good catch! I updated this to make it more clear. Not the priority definitions (since I want that to be mirrored), but how to select the right level once you introduce context. I'll revise this again after I walk the team through it. Prioritization is more of an art and I think walking them through this next week will give me loads more updates to make to make this even more clear! I also call this out in the support workflow document, so please consider this current state :)